### PR TITLE
400 KafkaRedis source/sink adapters

### LINKED ISSUE #400: KafkaRedis source/sink adapters
Implement source and sink adapters for kafka.

Source adapter subscribes to kafka topic, receives frames and sends to the module. When the frame content is in Redis the adapter fetches it and also sends to the module.

Sink adapter receives frames from the module and sends them to kafka topic, if Redis is configured, the frames are sent to Redis, metadata are sent to KAfka.

Add a sample `VideoLoop -> Kafka Sink -> Kafka/KeyDB -> Kafka Src -> Mod -> AO-RTSP`.

### PR DESCRIPTION
**Pull request by tomskikh**

### CODE REVIEW

File: samples/kafka_redis_adapter/docker-compose.l4t.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    volumes:
+      - zmq_sockets:/tmp/zmq-sockets
+    environment:
+      - ZMQ_ENDPOINT=sub+connect:ipc:///tmp/zmq-sockets/loop-source.ipc
+      - KAFKA_BROKERS=kafka:9092
+      - KAFKA_TOPIC=kafka-reds-adapter-demo
+      - KAFKA_CREATE_TOPIC=True
+      - 'KAFKA_CREATE_TOPIC_CONFIG={"num_partitions": 4, "replication_factor": 1}'
+      - REDIS_HOST=keydb
+      - REDIS_PORT=6379
```
Comment (bwsw) [inline samples/kafka_redis_adapter/docker-compose.l4t.yml:66]: @tomskikh Need TTL setting

File: samples/kafka_redis_adapter/docker-compose.l4t.yml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+  kafka-redis-sink:
+    image: ghcr.io/insight-platform/savant-adapters-py-l4t:latest
+    restart: unless-stopped
+    volumes:
+      - zmq_sockets:/tmp/zmq-sockets
+    environment:
+      - ZMQ_ENDPOINT=sub+connect:ipc:///tmp/zmq-sockets/loop-source.ipc
+      - KAFKA_BROKERS=kafka:9092
+      - KAFKA_TOPIC=kafka-reds-adapter-demo
```
Comment (bwsw) [inline samples/kafka_redis_adapter/docker-compose.l4t.yml]: @tomskikh need ttl for Kafka & Redis (separate)

File: samples/kafka_redis_adapter/docker-compose.l4t.yml (hunk 1 above)
Comment (bwsw) [reply samples/kafka_redis_adapter/docker-compose.l4t.yml:66]: What about Redis DB?

Comment (tomskikh) [reply samples/kafka_redis_adapter/docker-compose.l4t.yml:66]: Added TTL and DB settings to the sample.

File: samples/kafka_redis_adapter/docker-compose.l4t.yml (hunk 2 above)
Comment (tomskikh) [reply samples/kafka_redis_adapter/docker-compose.l4t.yml]: Added TTLs to the sample.

File: savant/client/__init__.py [hunk 3]
```diff
@@ -3,6 +3,12 @@
...
-from .frame_source.jpeg import JpegSource
 from .log_provider import LogProvider
 from .log_provider.jaeger import JaegerLogProvider
+
+try:
+    from .jpeg_source import JpegSource
+except ImportError:
+    import logging
+
+    logging.warning('Failed to import JpegSource. Install OpenCV to enable it.')
```
Comment (abramov-oleg) [inline savant/client/__init__.py]: Why root logging and not logger object with 'insight.savant' prefix?

Comment (tomskikh) [reply savant/client/__init__.py]: Changed the logger.

**Review by abramov-oleg — APPROVED** [review]

**Pull request merged**